package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
     "sass": "^1.32.8",
+    "styled-components": "^5.2.3",
```

Comment:
Choose one or the other

Reply:
remove scss/all files are in styled-components